app/components/container/containerController.js

```diff
@@ -117,22 +117,29 @@ function ($scope, $state, $stateParams, $filter, Container, ContainerCommit, Ima
   };
 
   $scope.confirmRemove = function () {
+    var title = 'You are about to remove a container.';
     if ($scope.container.State.Running) {
-      ModalService.confirmDeletion(
-        'You are about to remove a running container.',
-        function (confirmed) {
-          if(!confirmed) { return; }
-          $scope.remove();
+      title = 'You are about to remove a running container.';
+    }
+    //var text = '<label for="clean" class="control-label text-left">Clean associated volumes<portainer-tooltip position="bottom" message="Enable this option to remove associated volumes."></portainer-tooltip></label><label class="switch" style="margin-left: 20px;"><input type="checkbox"><i></i></label>';
+    var text = '<label for="clean" class="control-label text-left">Automatically clean associated volumes</label><label class="switch" style="margin-left: 21px;"><input type="checkbox"><i></i></label>';
+    ModalService.confirmContainerDeletion(
+      title,
+      text,
+        function (result) {
+          if(!result) { return; }
+          var cleanAssociatedVolumes = false;
+          for (var i in result) {
```

Comment:
Ah, OK. I see, I really don't like this solution. Would it be possible to have only one checkbox (generated by bootbox using `inputOptions` and change the css of that checkbox ? See my comment below regarding css rules.